src/file/stat.windows.cr

```diff
@@ -0,0 +1,142 @@
+class File
+  struct Stat
+    def initialize(filename : String)
+      if LibC.stat(filename, out @stat) != 0
+        raise Errno.new("Unable to get stat for '#{filename}'")
+      end
+    end
+
+    def initialize(@stat : LibC::Stat)
+    end
+
+    def atime
+      {% if flag?(:darwin) %}
```

Comment:
A lot of the Windows code is simply not runable at all, it was copied from the original code simply to get it to compile. The `stat` struct and function doesn't exist on windows and this will be to be rewritten using specific API. Right now, it is known to be broken :)